packages/DataTable/stories/MemorizedNavigation.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import DataTable, { Filters, Sort, ColumnManaging } from "../src";
+import fixtures from "./fixtures";
+import { viewPortHeight } from "./helpers";
+
+const mockData = fixtures(1);
+function App() {
+  return (
+    <DataTable localStorageId="my-table-id" keygen="id" data={mockData} height={viewPortHeight()}>
```

Comment:
I don't think we should force the user to create localStorageId, this is the same con ide on the ColumnDefinition, honestly I think those should be handle by our selves, is quite simple to create a `nanoid` in DataTable, and let our consumer don't worry about this.
```js
  import nanoid from "nanoid"
  const id = React.useState(nanoid)
```

Reply:
mmm wait, but then wouldn't be possible to know which table to retrive froom the localstorage because can be multiple tables, ok. maybe let's change th ename to `<DataTable id="" />`

And let's eventually remove the requirement from adding a `id` on Column definition.